qualtricssurvey/mixins/fragment.py

```diff
@@ -51,6 +59,36 @@ def student_view(self, context=None):
             js=static_js,
             js_init=js_init,
         )
+    
+        # Checking if the survey has subscription for event callback and creates it if not
+        try:
+            qualtrics_subscription = QualtricsSubscriptions.objects.get(course_id=getattr(self.runtime, 'course_id', None), usage_key=self.location)
+    
+        except:
+            headers = {'X-API-TOKEN': u'{}'.format(settings.QUALTRICS_API_TOKEN), 'Content-Type': 'application/json'}
+            payload = json.dumps({
+                "topics": "surveyengine.completedResponse." + self.survey_id,
+                "publicationUrl": "http://google.com/courses/course-v1:edX+DemoX+Demo_Course/xblock/block-v1:edX+DemoX+Demo_Course+type@qualtricssurvey+block@00116206cd3a4059b4749fe26b5417bd/handler_noauth/end_survey"
+            })
```

Comment:
@bsande6 The `publicationUrl` payload value needs to be unique and it should point back to the specific XBlock event handler address. It looks like you hard-coded this but didn't update it.

Reply:
Handled this with https://github.com/CUCWD/xblock-qualtrics-survey/pull/6/commits/c1f00430b50ab3727a50fd333455c9500bc92156

Reply:
Handled update with https://github.com/CUCWD/xblock-qualtrics-survey/commit/6c1017df7b3e50f120bc7829cc32abf84adf05a0